src/WebUI/Components/ListComponent.scss

```diff
@@ -4,16 +4,18 @@
     padding: 16px;
     border-radius: 4px;
     /* todo :: use std colors */
-    background-color: white;
+    background-color: $white;

```

Comment:
this is causing some issue with accessibility, check the bug #65 , we may have to use -- notation in css, so that they are filled on the fly.